src/conn/conn_api.c

```diff
@@ -1359,6 +1351,14 @@ __conn_close(WT_CONNECTION *wt_conn, const char *config)
     WT_TRET(__wt_checkpoint_cleanup_destroy(session));
     WT_TRET(__wti_checkpoint_server_destroy(session));
 
+    /*
+     * Shut down the layered table manager thread, ideally this would be taken care of in connection
+     * close below, but it needs to precede global transaction state shutdown, so do it here as
+     * well. It also needs to happen prior to draining transaction activity - since the manager
+     * artificially pins transaction state to allow for garbage collection in ingest tables.
```

Comment:
We should extend this comment again, noting the interaction with the sweep server.